src/app/shared/components/staff-record-summary/qualifications-and-training/qualifications-and-training.component.html

```diff
@@ -103,16 +87,12 @@
       </app-summary-record-value>
     </dd>
     <dd class="govuk-summary-list__actions">
```

Comment:
```suggestion
    <dd *ngIf="canEditWorker" class="govuk-summary-list__actions">
```
Do we need to add the `canEditWorker` permission here?

Reply:
Good catch, it should be in the dd not the a.